pkg/models/reading.go

```diff
@@ -26,34 +26,28 @@ import (
  * Struct for the Reading object in EdgeX
  */
 type Reading struct {
-	Id       string `json:"id"`
-	Pushed   int64  `json:"pushed"`  // When the data was pushed out of EdgeX (0 - not pushed yet)
-	Created  int64  `json:"created"` // When the reading was created
-	Origin   int64  `json:"origin"`
-	Modified int64  `json:"modified"`
-	Device   string `json:"device"`
-	Name     string `json:"name"`
-	Value    string `json:"value"` // Device sensor data value
+	BaseObject
```

Comment:
Not sure we should be adding BaseObject back in here. We want to get rid of it, as discussed, and I think a previous PR refactored this type to remove it.

Reply:
Corrected.